### DISCUSSION TITLE
How to manually install winget-cli on Win 10 or 11?

### DISCUSSION THREAD

**Discussion by MarGeb1** [microsoft/winget-cli#2544] [Q&A] [answered]
Hi there, when I try to manually install winget-cli on Win 10 or 11 it tells me, that it can't be installed via APP-Installer because specific restricted functions are used.

What am I doing wrong?

Markus

**Comment by MarGeb1** [comment]
No one? Really??

**Reply by mdanish-kh** [reply]
cc @denelon

**Comment by denelon** [comment, marked as answer]
@MarGeb1 Is this machine able to access the Microsoft Store to install the "App Installer"? That's the package the Windows Package Manager is distributed with.

When you state "manually install", what are you referring to? We have instructions on the main README at winget-cli for dependencies, and we have added the license file in the artifacts for GitHub releases in this repository as well.

**Reply by MarGeb1** [reply]
Hi @denelon, I want to preinstall machines with sysprep and it's very import not to use the MS-Store, because installing or actualizing Apps often causes sysprep to fail.

That's why I always installed the msixbundle manually with a simple double-click. But that doesn't work with newer builds, the message that it can't be installed via APP-Installer because specific restricted functions are used shows up.

But at this moment with another test I found out that winget is enabled nevertheless after this error message and that is okay for me.

Thank you anyway for your help.

Markus
